linker/Tests/Mono.Linker.Tests.Cases.Expectations/Assertions/KeptMemberInAssemblyAttribute.cs

```diff
@@ -5,13 +5,20 @@ namespace Mono.Linker.Tests.Cases.Expectations.Assertions {
 	[AttributeUsage (AttributeTargets.Class | AttributeTargets.Delegate, AllowMultiple = true, Inherited = false)]
 	public class KeptMemberInAssemblyAttribute : BaseExpectedLinkedBehaviorAttribute {
 		public readonly string AssemblyFileName;
```

Comment:
I've been on the fence about what to do as well.  My concern with dropping the fields is that then the constructor arguments will be highlighted in the IDE as not used and that would be annoying in it's own way.